src/fiatExchanges/utils.tsx

```diff
@@ -284,15 +284,15 @@ export const filterLegacyMobileMoneyProviders = (
 
 export async function fetchExchanges(
   countryCodeAlpha2: string | null,
-  tokenId: string
+  tokenId?: string
 ): Promise<ExternalExchangeProvider[] | undefined> {
   // If user location data is not available, default fetching exchanges serving the US
   if (!countryCodeAlpha2) countryCodeAlpha2 = 'us'
   // Standardize cGLD to CELO
 
   try {
     const resp = await fetchWithTimeout(
-      `${networkConfig.fetchExchangesUrl}?country=${countryCodeAlpha2}&tokenId=${tokenId}`,
+      `${networkConfig.fetchExchangesUrl}?country=${countryCodeAlpha2}${tokenId ? `&tokenId=${tokenId}` : ''}`,
```

Comment:
What's the result when no tokenId is provided?

Reply:
thanks for calling this out! i did a (too) quick investigation today and saw that the `getExchanges` CF marks `tokenId` as optional. i didn't dig further, expecting that the CF would then return only exchanges based on the provided query params...but it seems like the CF does default to CELO if the tokenId is not provided 😅 i think for this use case, we can't show the exchanges unless we add token selection first. i see the exchanges part isn't in the design anyway so for now i removed it